applications/kommander/0.16.0/defaults/cm.yaml

```diff
@@ -128,5 +128,11 @@ data:
         - "kommander-flux"
         - "cilium-hubble-relay-traefik"
     catalogCollections:
-    - registry: ghcr.io
-      repository: nutanix-cloud-native/nkp-partner-catalog
+    - resourceURI: oci://ghcr.io/nutanix-cloud-native/nkp-partner-catalog/collection
+      catalogName: nkp-partner-catalog
+      tag: "" # Defaults to MAJOR.MINOR of KommanderCore when empty.
+      extraLabels: {}
+    - resourceURI: oci://ghcr.io/nutanix-cloud-native/nkp-nutanix-product-catalog/collection
+      catalogName: nkp-nutanix-product-catalog
+      extraLabels: {}
+      tag: "" # Defaults to MAJOR.MINOR of KommanderCore when empty.
```

Comment:
These do not exist yet and will result in OCIRepositories that are in failing state but that should not affect any other functionality.